src/contrib/persistence/Akka.Persistence.Query.Sql/AllPersistenceIdsPublisher.cs

```diff
@@ -169,7 +170,7 @@ protected override bool Receive(object message)
                 case Request _:
                     Become(Waiting);
                     _journalRef
-                        .Ask<CurrentPersistenceIds>(new SelectCurrentPersistenceIds(_lastOrderingOffset, Self))
+                        .Ask<CurrentPersistenceIds>(new SelectCurrentPersistenceIds(_lastOrderingOffset, Self), DefaultQueryTimeout)
```

Comment:
Added default timeouts on all `Ask<T>` operations to the throttler / journal in order to make queries responsive in the event that the journal fails.